tools/backup-setup/config/backup.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: ark.heptio.com/v1
+kind: Backup
+metadata:
+  name: kyma-backup
```

Comment:
This should be named customer-namespaces-backup or something like this. Please create a second file for system namespaces. Some teams e.g. gophers need to backup etcd in kyma-system namespace.